.github/scripts/set-user-agent.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+printf '\nset user agent\n\n'
+
+FILE=Sources/WalletConnectRelay/PackageConfig.json
+
+if [ -f "$FILE" ];
+then
+    printf '\ncurrent user agent:\n'
+    cat "$FILE"
+    printf '\nsetting user agent... \n'
+    echo "{\"version\": \"$PACKAGE_VERSION\"}" > "$FILE"
```

Comment:
this will replace current value? Or just appending?

Reply:
replace